msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -233,6 +243,12 @@ Upon first run of local debug to Office or Outlook, you'll be prompted to sign i
 
 Provide feedback and report any issues with the Teams Toolkit debugging experience at [Microsoft Teams Framework (TeamsFx)](https://github.com/OfficeDev/TeamsFx/issues).
 
+#### Mobile debugging
+
+1. You can remotely debug your app in Office for Android. From your profile **(Me) > Settings > Allow debugging**, toggle on the option for **Enable remote debugging**.
```

Comment:
setting name is Enable remote debugging instead of allow debugging.

Reply:
*Enable remote debugging* toggle is under the header for "Allow debugging" in settings menu